boards/riscv/hpm6750evkmini/support/soc/hpm6750-dual-core.cfg

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2022 HPMicro
```

Comment:
Where is this file used? Because it doesn't seem to be passed in `board.cmake` to OpenOCD?

Reply:
Set aside for dual-core scenarios. Plan to adapt to dual-core, appear here a little early, delete.

Reply:
Doesn't seem to be deleted?